src/app/conf/2025/components/become-a-sponsor/index.tsx

```diff
@@ -0,0 +1,125 @@
+import clsx from "clsx"
+
+import { Button } from "../../../_design-system/button"
+
+import blurBlob from "./blur-blob.webp"
+import { StripesDecoration } from "@/app/conf/_design-system/stripes-decoration"
+
+export function BecomeASponsor() {
+  return (
+    <section id="sponsors" className="relative">
+      <Stripes />
+      <div className="gql-conf-container gql-conf-section xl:py-16 2xl:px-64">
```

Comment:
so we can make the section same width as others even on larger screens.

```suggestion
      <div className="gql-conf-container gql-conf-section xl:py-16">
```

| Before | After | 
| -------|-------|
| ![CleanShot 2025-05-20 at 00 06 50@2x](https://github.com/user-attachments/assets/50340066-9297-4bcb-b7ef-0d0f0ed5a4fb) |  ![CleanShot 2025-05-20 at 00 06 34@2x](https://github.com/user-attachments/assets/3625c6a1-d317-41b5-9e9b-061e14f4944f) |